js/package.json

```diff
@@ -31,12 +31,11 @@
   },
   "devDependencies": {
     "@jupyterlab/builder": "^3.0.0",
-    "rimraf": "^2.6.1",
     "webpack": "^5",
-    "webpack-cli": "^4.7.2"
+    "rimraf": "^2.6.1"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1 || ^2 || ^3 || ^4",
+    "@jupyter-widgets/base": "^1.1 || ^2 || ^3 || ^4 || ^6",
```

Comment:
Is 5 intentionally not listed?

Reply:
yes, but you are right we can add sup to 5

"Note that “@jupyter-widgets/base” version 5 is reserved for ipywidgets 7 support on JupyterLab 4, “@jupyter-widgets/base” version 6 is the version released with ipywidgets 8."

https://ipywidgets.readthedocs.io/en/latest/migration_guides.html

Reply:
Just do what you think works best. I'm not that familiar with the Jupyter landscape.